ipa325_wsg50/urdf/wsg50.urdf.xacro

```diff
@@ -32,13 +32,37 @@
             </collision>
         </link>
 
-        <joint name="${prefix}jaw_base" type="fixed" >
+        <joint name="${prefix}finger_base_joint" type="fixed" >
             <origin xyz="0.0 0.0 0.0725" rpy="0 0 0" />
             <parent link="${prefix}wsg_base" />
-            <child link="${prefix}jaw_base" />
+            <child link="${prefix}finger_base" />
         </joint>
 
-        <link name="${prefix}jaw_base"/>
+        <link name="${prefix}finger_base"/>
+
+        <joint name="${prefix}finger_left_joint" type="prismatic">
```

Comment:
```suggestion
        <joint name="${prefix}finger_left" type="prismatic">
```
we typically use `finger_left` for the joint and `finger_left_frame` for the link but I'll check if that is standardized somewhere

Reply:
@simonschmeisser any update here?

Reply:
Note that `finger_left` is the default in WSG50ROSNode.cpp line 52 or so